modules/verifybamid/verifybamid2/meta.yml

```
@@ -0,0 +1,84 @@
+name: "VERIFYBAMID_VERIFYBAMID2"
+description: Detecting and estimating inter-sample DNA contamination became a crucial quality assessment step to ensure high quality sequence reads and reliable downstream analysis.
+keywords:
+  - contamination
+  - bam
+tools:
+  - "verifybamid2":
+      description: "A robust tool for DNA contamination estimation from sequence reads using ancestry-agnostic method."
+      homepage: "http://griffan.github.io/VerifyBamID"
+      documentation: "http://griffan.github.io/VerifyBamID"
+      tool_dev_url: "https://github.com/Griffan/VerifyBamID"
+      doi: "https://doi.org/10.1101/gr.246934.118"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - SVDPrefix:
+      type: map
+      description: |
+        SVD related files prefix [Required if RefVCF is not present]
+        e.g parental path shared by .UD, .mu and .bed files
+  - BamFile:
+      type: file
+      description: BAM/CRAM/SAM file
+      pattern: "*.{bam,cram,sam}"
+  - Reference:
+      type: file
+      description: reference file [Required]
+      pattern: "*.fasta"
+  ## To construct SVDPrefix auxillary files
+  - RefVCF:
+      type: file
+      description: Reference panel VCF with genotype information, for generation of .UD .mu .bed files [Optional]
+  - UDPath:
+      type: file
+      description: .UD matrix file from SVD result of genotype matrix
+  - MeanPath:
+      type: file
+      description: .mu matrix file of genotype matrix
+  - BedPath:
+      type: file
+      description: .Bed file for markers used in this analysis,format(chr\tpos-1\tpos\trefAllele\taltAllele)[Required]
```

```suggestion
  - UDPath:
      type: file
      description: .UD matrix file from SVD result of genotype matrix
  - MeanPath:
      type: file
      description: .mu matrix file of genotype matrix
  - BedPath:
      type: file
      description: .Bed file for markers used in this analysis,format(chr\tpos-1\tpos\trefAllele\taltAllele)[Required]
```

Check the names of these channels. Also make sure they are all lower case matching the input channel names